docs/modules/operation/pages/provisioning/scalability.adoc

```diff
@@ -1,42 +1,42 @@
+The days of writing scripts or single threaded applications no longer is sufficient to do the work required by a monitoring system at the scale of most networks today.
 
-== Parallelization and Non-Blocking I/O
+== Parallelization and non-blocking I/O
 
-Squeezing out every ounce of power from a management system’s platform (hardware and OS) is absolutely required to complete all the work of a fully functional NMS such as {page-component-title}.
-Fortunately, the hardware and CPU architecture of a modern computing platform provides multiple CPUs with multiple cores having instruction sets that include support for atomic operations.
-While these very powerful resources are being provided by commodity systems, it makes the complexity of developing applications to use them vs. not using them, orders of magnitude more complex.
-However, because of scalability demands of our complex IT environments, multi-threaded NMS applications are now essential and this has fully exposed the complex issues of concurrency in software development.
+Squeezing out every ounce of power from a management system’s platform is absolutely required to complete all the work of a fully functional monitoring system such as {page-component-title}.
+Fortunately, the hardware architecture of a modern computing platforms provides multiple CPUs with multiple cores having instruction sets that include support for atomic operations.
+Because of scalability demands of complex IT environments, multi-threaded monitoring solutions are now essential and this has fully exposed the complex issues of concurrency in software development.
 
-{page-component-title} has stepped up to this challenge with its new concurrency strategy.
-This strategy is based on a technique that combines the efficiency of parallel (asynchronous) operations (traditionally used by most effectively by single threaded applications) with the power of a fully current, non-blocking, multi-threaded design.
-The non-blocking component of this new concurrency strategy added greater complexity but {page-component-title} gained orders of magnitude in increased scalability.
+OpenNMS has stepped up to this challenge with its concurrency strategy.
+This strategy is based on a technique that combines the efficiency of parallel (asynchronous) operations, traditionally used by most effectively by single threaded applications, with the power of a fully current, non-blocking, multi-threaded design.
+The non-blocking component of this concurrency strategy adds greater complexity, but gives {page-component-title} substantially increased scalability.
 
-NOTE: Java Runtimes, based on the Sun JVM, have provided implementations for processor based atomic operations and is the basis for {page-component-title}’ non-blocking concurrency algorithms.
+NOTE: Java Runtimes, based on the Sun JVM, have provided implementations for processor-based atomic operations, and is the basis for {page-component-title}’s non-blocking concurrency algorithms.
```

Comment:
```suggestion
NOTE: Java Runtime, based on the Sun JVM, provides implementations for processor-based atomic operations, and are the basis for {page-component-title}’s non-blocking concurrency algorithms.
```
Not sure if there is a reason this was plural.

Reply:
There are multiple versions and flavors of Java available (Oracle, OpenJDK), but in modernizing this page, I don't think we lose anything by omitting this sentence.
```suggestion
```